web/test/factories.ts

```diff
@@ -722,26 +722,37 @@ export const randomOwnershipType = (): OwnershipType => {
   return ownershipTypes[randomIndex];
 };
 
-export const randomIndustry = (canHavePermanentLocation = false): Industry => {
+export const randomFilteredIndustry = (func: (industry: Industry) => boolean, enabled = true): Industry => {
```

Comment:
This is feedback I gave to Soma last week and I want to start enforcing as much as possible - we should try to never have raw booleans in function parameters. It's pretty confusing to see something like `randomFilteredIndustry(func, false)` and have no idea what that false represents. Anytime we have a boolean parameter it should be inside of an object so that it can be a named parameter like `randomFilteredIndustry(func, {isEnabled: false})` which is much clearer